xc/device_common.ml

```diff
@@ -160,6 +160,9 @@ let get_private_path_by_uuid uuid =
 let get_private_data_path_of_device (x: device) =
 	sprintf "%s/private/%s/%d" (get_private_path x.frontend.domid) (string_of_kind x.backend.kind) x.backend.devid
 
+let string_uuid_of_domid domid =
+	sprintf "%s" (uuid_of_domid domid)	
+
```

Comment:
Why not add `uuid_of_domid` to the `Device_common` interface? This function doesn't look like it does anything beyond what `uuid_of_domid` already does.